scripts/common/rook.sh

```diff
@@ -252,7 +252,7 @@ function maybe_cleanup_rook() {
                 report_addon_fail "rook-ceph-removal" "v1"
                 return
             fi
-            report_addon_success "rook-ceph-removal" "v1"
+            report_addon_success "rook-ceph-removal" "v1.1"
```

Comment:
All three of these need to be changed - lines 249 and 252

Reply:
Thanks now I got.
We will have a new version of the report after it get merged ,

Reply:
why the version bump?